scripts/expected_doxygen_warnings.txt

```diff
@@ -15,26 +15,11 @@
-  parameter 'get_string_expr'
-/cbmc/src/solvers/refinement/string_refinement.cpp:1511: warning: argument 'stream' of command @param is not found in the argument list of compute_inverse_function(const exprt &qvar, const exprt &val, const exprt &f)
 /cbmc/src/solvers/refinement/string_refinement.cpp:239: warning: Invalid list item found
-/cbmc/src/solvers/refinement/string_refinement.cpp:1823: warning: argument 'stream' of command @param is not found in the argument list of instantiate(const string_constraintt &axiom, const exprt &str, const exprt &val)
-/cbmc/src/solvers/refinement/string_refinement.cpp:1052: warning: The following parameters of substitute_array_access(const with_exprt &expr, const exprt &index, const bool left_propagate) are not documented:
-  parameter 'left_propagate'
 /cbmc/doc/architectural/howto.md:260: warning: found </div> at different nesting level (6) than expected (3)
 /cbmc/doc/architectural/howto.md:261: warning: end of comment block while expecting command </div>
-/cbmc/src/solvers/README.md:4: warning: @copydetails or @copydoc target 'generate_instantiations(messaget::mstreamt &stream, const string_constraint_generatort &generator, const index_set_pairt &index_set, const string_axiomst &axioms, const std::map<string_not_contains_constraintt, symbol_exprt> &not_contain_witnesses).' not found
-/cbmc/src/cbmc/bmc.cpp:473: warning: argument 'message' of command @param is not found in the argument list of bmct::do_language_agnostic_bmc(const path_strategy_choosert &path_strategy_chooser, const optionst &opts, abstract_goto_modelt &model, ui_message_handlert &ui, std::function< void(bmct &, const symbol_tablet &)> driver_configure_bmc=nullptr, std::function< bool(void)> callback_after_symex=nullptr)
-/cbmc/src/cbmc/bmc.h:124: warning: The following parameters of bmct::do_language_agnostic_bmc(const path_strategy_choosert &path_strategy_chooser, const optionst &opts, abstract_goto_modelt &model, ui_message_handlert &ui, std::function< void(bmct &, const symbol_tablet &)> driver_configure_bmc=nullptr, std::function< bool(void)> callback_after_symex=nullptr) are not documented:
-  parameter 'path_strategy_chooser'
-/cbmc/src/solvers/flattening/bv_utils.cpp:1113: warning: argument 'Bit' of command @param is not found in the argument list of bv_utilst::equal(const bvt &op0, const bvt &op1)
-/cbmc/src/solvers/flattening/bv_utils.cpp:1113: warning: argument 'vectors' of command @param is not found in the argument list of bv_utilst::equal(const bvt &op0, const bvt &op1)
-/cbmc/src/solvers/flattening/bv_utils.h:134: warning: The following parameters of bv_utilst::equal(const bvt &op0, const bvt &op1) are not documented:
-  parameter 'op0'
-  parameter 'op1'
+/cbmc/src/solvers/README.md:434: warning: Invalid list item found
```

Comment:
How come we have a new warning appearing?

Reply:
Unknown, appears to have been previously somehow hidden by the other warning in that file.

Reply:
This is pretty weird: that file doesn't even have 434 lines!?

Reply:
It's because of the use of the Doxygen `\copydetails` command in `solvers/README.md`, and the same problem is reported for the copied file:
`/cbmc/src/solvers/refinement/string_refinement.cpp:239: warning: Invalid list item found`

Reply:
That warning itself I couldn't quite figure out though, hence omitting it from this PR